packages/components/buttons/flat-button/src/flat-button.tsx

```diff
@@ -121,6 +121,7 @@ const FlatButton = <TStringOrComponent extends ElementType = 'button'>(
       css={css`
         min-height: initial;
         align-items: center;
+        font-size: ${designTokens.fontSizeForTextAsBody};
```

Comment:
Instead of having a new CSS rule here we could change the `span` which is wrapping button's text with a `Text.Body` component for better composability.

```
// BEFORE
<span>{props.label}</span>

// AFTER
<Text.Body as="span">{props.label}</Text.Body>
```

Reply:
Thanks. SO much better! 🧘‍♂️